contracts/sol6/KyberNetwork.sol

```diff
@@ -956,6 +974,7 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
                 reservesData.srcAmounts[i],
                 block.number
             );
+            //* reservesData.rates[i] <= MAX_RATE?  later validation when calc destAmount
```

Comment:
//* reservesData.rates[i] <= MAX_RATE. validated in validateTradeCalcDestQtyAndFeeData when calcDstQty is called